core/static/js/dit.class.modal.js

```diff
@@ -132,14 +137,28 @@
       }
     });
 
+    self.$firstLink.on("keydown", function(e) {
```

Comment:
Ok, I see what you're doing with the $firstLink.on and additions to $closeButton events. 
We should be doing this different though. 

In Modal.prototype.focus we're getting the first focusable element. We should move this...
`self.$content.find("video, a, button, input, select").eq(0)`

...to within the constructor, and assign the return value (jQuery node reference) to an instance property, like you've done in `this.$firstLink`. It should obviously be renamed to something like $firstFocusElement, because it is not always going to be a link.

We then apply this on[keydown] event hander to that element, much like you have done., but it should not happen inside a class function called `Modal.bindCloseEvents `- that's simply confusing the code organisation.

Perhaps we should have some `Modal.bindKeyboardFocusEvents`, or something like that.